src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
Let's assume that DXCoil( DXCoilNum ).ElecHeatingPower is calculated correctly for now. The only other thing that might need checking is crankcase heater power, based on RTF, which is based on PLRHeating. Again, how to resolve these as being correctly calculated.